dev-client/src/components/StaticMapView.tsx

```diff
@@ -111,7 +111,7 @@ export const StaticMapView = ({
   const cameraSettings = useMemo(
     () =>
       ({
-        centerCoordinate: coordsToPosition(coords),
+        centerCoordinate: coords ? coordsToPosition(coords) : undefined,
```

Comment:
Question: What were the errors caused by this? I'm confused because `coords` is defined as the `Coords` type, which (as far as I know) can't be undefined/null/falsy. So I'd expect this change to not really have any effect, but maybe I'm missing something?

Reply:
This seems to happen with hot reloads when building/debugging in the simulator. It's possible/likely there is a better way to fix this.